tests/test_xopen.py

```diff
@@ -392,6 +492,14 @@ def test_truncated_gz(truncated_gzip):
             f.close()  # pragma: no cover
 
 
+def test_truncated_bz2(truncated_bz2):
```

Comment:
To eliminate the duplication, it would be nice to switch to a `truncated_compressed_file` fixture or something like that.